prow/github/types.go

```diff
@@ -215,21 +215,32 @@ type Branch struct {
 	Protected bool   `json:"protected"`
 }
 
-// https://developer.github.com/v3/repos/branches/#update-branch-protection
+// BranchProtectionRequest restricts who can write to a branch.
+// All keys required. Disable options by passing a nil value.
+//
+// See: https://developer.github.com/v3/repos/branches/#update-branch-protection
 type BranchProtectionRequest struct {
-	RequiredStatusChecks       RequiredStatusChecks        `json:"required_status_checks"`
-	EnforceAdmins              bool                        `json:"enforce_admins"`
-	RequiredPullRequestReviews *RequiredPullRequestReviews `json:"required_pull_request_reviews"`
-	Restrictions               Restrictions                `json:"restrictions"`
+	EnforceAdmins               *bool                                  `json:"enforce_admins"` // null to disable
```

Comment:
Omitempty so when unset is `nil`?

Reply:
I explicitly want to send `"enforce_admins": null` per github docs

https://developer.github.com/v3/repos/branches/#update-branch-protection